packages/content/style.module.css

```diff
@@ -0,0 +1,27 @@
+.root {
```

Comment:
I went with a pretty straightforward conversion here, and put a bit of extra effort into paving the way for a more re-usable approach.

This approach has already come in handy in removing the large overlap between revised `blog` styles, which were  largely duplicative of this component's previous styles, apart from headings which design had tweaked. See https://github.com/hashicorp/hashicorp-www-next/pull/2100 for details.

Apart from greater modularity, this method of organization is also intended to 1) make things a little easier to skim and dive into, and 2) prep for a more MDX-custom-component-based approach which we'll likely adopt for Docs and Learn.